views/dugong.erb

```diff
@@ -0,0 +1,76 @@
+<html>
+  <head>
+    <title>Dugong</title>
+    <meta charset = "utf-8">
+  </head>
+
+  <body>
+    <h1> Dugong</h1>
+
+    <table border="1" style="border-collapse: collapse; text-align: center">
+      </tbody>
+        <tr>
+          <td align = "left">
```

Comment:
In Ruby, we add spaces on either side of our `=`, but in HTML, we try to avoid those extra spaces